chain/types/ethtypes/eth_transactions.go

```diff
@@ -58,7 +58,44 @@ type EthTxArgs struct {
 	S                    big.Int     `json:"s"`
 }
 
-func EthTxArgsFromMessage(msg *types.Message) (EthTxArgs, error) {
+// EthTxFromSignedEthMessage does NOT populate:
+// - BlockHash
+// - BlockNumber
+// - TransactionIndex
+// - From
+// - Hash
+func EthTxFromSignedEthMessage(smsg *types.SignedMessage) (EthTx, error) {
```

Comment:
We kind of need some way of differentiating between `types.Message`/`types.SignedMessage` that can be converted into a legitimate `EthTx` or not. It makes sense to me to use `Transaction`/`Tx` for the Ethereum shape, and `Message` for the Filecoin shape, but I know other people might have differing thoughts about those words.

I just want to avoid the confusion of someone thinking they can use this function for any signed message.